content/03-reference/01-tools-and-interfaces/02-prisma-client/055-case-sensitivity.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: 'Case sensitivity'
+metaTitle: 'Case sensitivity (Reference)'
+metaDescription: 'How Prisma Client handles case sensitivity when filtering and sorting.'
+preview: false
+---
+
+<TopBlock>
+
+Case sensitivity affects **filtering** and **sorting** of data, and is determined by your [database collation](#database-collation-and-case-sensitivity). Sorting and filtering data yields different results depending on your settings:
+
+| Action          | Case sensitive                               | Case insensitive                             |
+|-----------------|----------------------------------------------|----------------------------------------------|
+| Sort ascending  | `Apple`, `Banana`, `apple pie`, `banana pie` | `Apple`, `apple pie`, `Banana`, `banana pie` |
+| Match `"apple"` | `apple`                                      | `Apple`, `apple`                             |
+
+[Prisma Client](.) respects your database collation. Options and recommendations for supporting **case-insensitive** filtering and sorting with Prisma Client depend on your [database provider](#options-for-case-insensitive-filtering).
```

Comment:
I do not think we support impacting the sorting yet: https://github.com/prisma/prisma-client-js/issues/841

Reply:
We don't, but it sorting is still affected - there's a link to the sorting GH issue on the page. Shall I add it to the first mention of sorting maybe?

Reply:
No opinion. Only skimmed the text really - if it makes sense in the big context, this is perfectly fine.